staging/src/k8s.io/apimachinery/pkg/util/proxy/dial.go

```diff
@@ -115,3 +129,15 @@ func DialURL(ctx context.Context, url *url.URL, transport http.RoundTripper) (ne
 		return nil, fmt.Errorf("Unknown scheme: %s", url.Scheme)
 	}
 }
+
+func supportsHTTP11(nextProtos []string) bool {
+	if nextProtos == nil {
+		return true
+	}
```

Comment:
Empty slice should return `true` as well right?

```go
	if len(nextProtos) == 0 {
		return true
	}
```

Reply:
I thought nextproto nil and empty slice were treated differently, but I can't find reference to that so I'm most likely wrong.

Reply:
updated